lib/google/gax/version.rb

```diff
@@ -29,6 +29,6 @@
 
 module Google
   module Gax
-    VERSION = '0.8.0'.freeze
+    VERSION = '0.9.0'.freeze
```

Comment:
Ah It escaped my mind that I purposefully made the http headers change backwards compatible. Will do.